HidInjector/driver/HidInjectorKd.c

```diff
@@ -287,7 +288,8 @@ Return Value:
 	// Initialize VHF.  This will talk to HIDCLASS for us.
 	//
 	status = HIDINJECTOR_VhfInitialize(device);
-	if (!NT_SUCCESS(status)) {
+	if (!NT_SUCCESS(status)) 
+    {
```

Comment:
Can you check the indents (ie. space vs tabs?)  On chrome, the indents are off, but they look fine in IE.  Not sure which one is correct.